packages/commons/src/auth/jwt.ts

```diff
@@ -59,14 +56,6 @@ export const readAuthDataFromJwtToken = (
   }
 };
 
-const clients = !config.skipJWTVerification
```

Comment:
Moved this inside `verifyJwtToken` to avoid the need to call `config = commonConfig()` outside a function - otherwise, we would require adding a `.env` config in `commons` when running the `commons` tests I added for file manager.